app/controllers/application_controller.rb

```diff
@@ -14,7 +14,7 @@ class ApplicationController < ActionController::API
   rescue_from Pundit::NotAuthorizedError, with: :user_not_authorized
   rescue_from ActiveRecord::RecordNotFound, with: :model_not_found
 
-  has_scope :sorted, default: nil, allow_blank: true
+  # has_scope :sorted, default: nil, allow_blank: true
```

Comment:
@chvp I had to tmp comment this line, since it was causing problems.
When the line was included our query also tried to order per id, which was not present in the `GROUP BY` clause. (Even if we didn't have any scopes). Do we still use/need this somewhere?
```
ERROR:  column "plays.id" must appear in the GROUP BY clause or be used in an aggregate function at character 188
STATEMENT:  SELECT COUNT(*), MAX(played_at) as last_played_at, "plays"."track_id", "plays"."user_id" FROM "plays" WHERE "plays"."user_id" = $1 GROUP BY "plays"."track_id", "plays"."user_id" ORDER BY "plays"."id" DESC, "plays"."track_id" DESC LIMIT $2 OFFSET $3
```

Reply:
Clients implicitly rely on sorting by id to handle the edge case where new data appears while syncing. It needs at least some ordering, otherwise the database might decide to start ordering differently, leaving the clients with some missed data (even if some data disappears). The reverse by id is the order where there is the least chance for older data to disappear.